linchpin/ansible_runner.py

```diff
@@ -50,7 +52,7 @@ def suppress_stdout():
 
 def ansible_runner_2x(playbook_path,
                       extra_vars,
-                      options=None,
```

Comment:
@14rcole Should this be removed in 2.x ? runner too ?

Reply:
No, you're right.  That's leftover from troubleshooting